bind_test.go

```
@@ -1180,13 +1223,69 @@
+		Name string `form:"name"`
+		Age  int    `form:"age"`
+	}
+
 	type Demo struct {
+		Name    string   `form:"name"`
+		Persons []Person `form:"persons"`
+	}
+
+	buf := &bytes.Buffer{}
+	writer := multipart.NewWriter(buf)
+	writer.WriteField("name", "john")
```

golangci-lint (1.62.2)

[warning] unhandled-error: Unhandled error in call to function mime/multipart.Writer.WriteField

(revive)